base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
What is the point of your argument? This appears to be a validation that midipix would is a viable counterpart to win32 as a api to the nt kernel, while wsl is an independent, disconnected kernel. License may be a concern for actually being able to use it, but it's not relevant to a discussion of what it is called.

It sounds like WSL may provide yet another cross-compile environment that we will be expected to support, without providing any benefit to end users. Since it lies about it's uname, this will likely be made more difficult, so forgive me if I'm not quite thrilled about this. cygwin is at least truthful about its uname and allows coding against both the win32 and posix APIs to create hybrid apps.